libjob_queue/src/ext_job.c

```diff
@@ -743,6 +748,40 @@ static void __fprintf_python_argList(FILE * stream,
 }
 
 
+static void __fprintf_python_arg_types(FILE * stream,
+                                       const char * prefix,
+                                       const char * key,
+                                       const ext_job_type * ext_job,
+                                       const char * suffix,
+                                       const char * null_value) {
+  fprintf(stream, "%s", prefix);
+  if (ext_job->arg_types) {
+    fprintf(stream, "\"%s\" : [", key);
+    int i;
+    for (i = 0; i < ext_job->max_arg; i++) {
```

Comment:
This should loop over the size of arg_types - which can very well be less than max_arg.

Reply:
I am not sure about that. arg_types should be set in the forward model config file and cannot be changed by the user (from the ert config file).

Reply:
The point is that you do not have to specify type for all the arguments, i.e. you can have the following situation:
```txt
MIN_ARG 1
MAX_ARG 3
ARG_TYPES 0 INT
```
In this case the function should have 1-3 arguments, the first argument should be an integer - *you do not care* about the two next arguments - meaning that they will be `CONFIG_STRING`.